docs/design/liqonet_tunEndCreator.md

```diff
@@ -0,0 +1,25 @@
+# TunnelEndpointCreator-Operator
+## Overview
+The TunnelEndpointCreator is part of the network module and runs as a deployment. It reacts to **Advertisements Custom Resources (CR)**
+and for each one it creates a **TunnelEndpoint CR**. The advertisement CR carries all the required data
+in order to establish a point-to-point connection with the peering cluster. A simple IP manager
+has been embedded inside the operator in order to resolve the possible conflicts rising between the network
+address spaces used by the clusters. The peering clusters could have address spaces conflicts:
+* pod network CIDR
+* clusterIP network CIDR
+
+### Features
+If the pod network CIDR of a peering cluster does not have conflicts with the address spaces
```

Comment:
What you write here is not a list of features. Move in the overview and add a list of features here.